src/Faker/Provider/es_BO/Address.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Faker\Provider\es_BO;
+
+class Address extends \Faker\Provider\es_ES\Address
+{
+    protected static $cityPrefix = array('');
+    protected static $citySuffix = array('');
+    protected static $buildingNumber = array('#####', '####', '###', '##', '#');
+    protected static $streetPrefix = array('Av.', 'Cl.', 'Urb.');
+    protected static $streetSuffix = array('');
+    protected static $postcode = array('BOLIVIA ####');
+    protected static $state = array(
+        'Santa Cruz', 'Beni', 'Pando', 'La Paz', 'Chuquisaca', 'Oruro', 'Tarija', 'Potosí', 'Cochabamba', 'El Alto'
+    );
+    protected static $stateAbbr = array(
+        'BO-SC', 'BO-LP', 'BO-BN', 'BO-PA', 'BO-CH', 'BO-OR', 'BO-TJ', 'BO-PT', 'BO-CO',
+    );
+    protected static $cityFormats = array(
+        '{{cityPrefix}} {{firstName}} {{lastName}}',
+        '{{cityPrefix}} {{firstName}}',
+        '{{firstName}} {{citySuffix}}',
+        '{{lastName}} {{citySuffix}}',
+    );
+    protected static $streetNameFormats = array(
+        '{{streetPrefix}} {{firstName}} {{lastName}}',
+    );
+    protected static $streetAddressFormats = array(
+        '{{streetName}} # {{buildingNumber}} ',
+        '{{streetName}} # {{buildingNumber}} {{secondaryAddress}}',
+    );
+    protected static $addressFormats = array(
+        "{{streetAddress}}\n{{city}}, {{state}}",
+    );
+    protected static $secondaryAddressFormats = array('Dpto. ###', 'Hab. ###', 'Piso #', 'Piso ##');
+
+    /**
+     * @example ''
```

Comment:
`@example` keywords are not really needed, please remove

Reply:
Ok, I will check it out as soon as possible